docs/layers/monitoring/datadog/datadog.mdx

```diff
@@ -69,133 +83,192 @@ Some definitions borrowed from Datadog's [https://docs.datadoghq.com/monitors/se
 Golden Signals are a form of telemetry that applies to anything with throughput. Anything that’s not a golden signal is considered general telemetry.
 
 <dl>
-  <dt>Latency</dt>
+  #### Latency
   <dd>The time it takes to service a request. It’s important to distinguish between the latency of successful requests and the latency of failed requests.</dd>
 
-  <dt>Traffic</dt>
+  #### Traffic
   <dd>A measure of how much demand is being placed on your system, measured in a high-level system-specific metric. For a web service, this measurement is usually HTTP requests per second.</dd>
 
-  <dt>Errors</dt>
+  #### Errors
   <dd>The rate of requests that fail, either explicitly (e.g., HTTP 500s), implicitly (for example, an HTTP 200 success response, but coupled with the wrong content), or by policy (for example, "If you committed to one-second response times, any request over one second is an error").</dd>
 
-  <dt>Saturation</dt>
+  #### Saturation
   <dd>How "full" your service is. A measure of your system fraction, emphasizing the resources that are most constrained (e.g., in a memory-constrained system, show memory; in an I/O-constrained system, show I/O).</dd>
 </dl>
```

Comment:
Use dt and dd tags. We should use headers for chapters and definition lists for that.